### DISCUSSION TITLE
Support for documentation comments (and/or attributes) for generated public property

### DISCUSSION THREAD

**Discussion by tkouba** [KafkaWannaFly/BindableProps#80] [Ideas]
I have this private property with documentation comment

```csharp
/// <summary>
/// Placeholder for text input
/// </summary>
[Localizable]
[BindableProp]
string placeHolder;
```

So I need to tranfser this comment and attribute also to generated public property.

I expect definition of public property like this

```csharp
/// <summary>
/// Placeholder for text input
/// </summary>
[Localizable]
public string PlaceHolder
{
  get => placeHolder;
  set 
  { 
  /* code removed */
  }
}
```

Is it possible to add this feature?

**Comment by KafkaWannaFly** [comment]
Frankly, I think no. Sorry 😗
In my opinion, that would cause doc comment duplicated and it's ugly. Not sure if any linting/code quality system would catch this bug.
Secondly, if u want a document for your lib user, that why u need to put it to public prop as I guess, u better have a document website as I or many popular frameworks do.
Thirdly, I feel this feature is just out of scope.
Anyway, thanks for supporting my product as a user 🫡

**Reply by tkouba** [reply]
This documentation comments are for IntelliSense, not for full library documentation. In private property should be only one-line comment, like this:
```csharp
// Placeholder for text input
[Localizable]
[BindableProp]
string placeHolder;
```
